s3_management/manage.py

```diff
@@ -146,7 +145,7 @@ def nightly_packages_to_show(self: S3IndexType) -> Set[str]:
         # also includes versions without GPU specifier (i.e. cu102) for easier
         # sorting, sorts in reverse to put the most recent versions first
         all_sorted_packages = sorted(
-            {self.normalize_package_version(obj) for obj in self.objects},
+            {self.normalize_package_version(s3_key) for s3_key in self.objects.keys()},
```

Comment:
`flake8-simplify` ([link](https://pypi.org/project/flake8-simplify/)) suggests just iterating over the dict instead of explicitly calling `.keys()`. We could talk about the likely very itty bitty performance bump, but lol.

Is there a reason you prefer `.keys()`?